addon/helpers/format-money.ts

```diff
@@ -1,6 +1,9 @@
 import Helper from '@ember/component/helper';
 
-var _getFormatter = function (currency: string, compact?: boolean) {
+const ANTI_COMPACT_RAW_THRESHOLD: number = 1000;
+const ANTI_COMPACT_CENTS_THRESHOLD: number = 100_000;
```

Comment:
Sorry but this took me a while to understand, how about
```
const PREVENT_COMPACT_NOTATION_ON_RAW_BELOW = 1000;
const PREVENT_COMPACT_NOTATION_ON_CENTS_BELOW = 100_000;
```

I know it's very long, but it leaves less room for misinterpretation IMO
😇